tests/commontransport.nim

```diff
@@ -137,6 +137,10 @@ proc commonTransportTest*(name: string, prov: TransportProvider, ma: string) =
       await transport1.stop()
 
     asyncTest "e2e should allow multiple local addresses":
+      when defined(windows):
+        # this randomly locks the Windows CI job
```

Comment:
Huh, maybe on windows it could use a timeout to allow stuff to cleanup between runs. Usually, issues that we've identified on windows (or other platforms) individually, have ended up being real problems.